src/ophyd_async/epics/_backend/_p4p.py

```diff
@@ -238,7 +238,6 @@ def __init__(self, datatype: Optional[Type[T]], read_pv: str, write_pv: str):
         self.converter: PvaConverter = DisconnectedPvaConverter()
         self.subscription: Optional[Subscription] = None
 
-    @property
```

Comment:
I don't think this should ever have been a property?

Reply:
nope, @jsouter's changes turned this into a method, so not sure how it ever worked with the property decorator?